Krypt/Source/AES256.swift

```diff
@@ -72,8 +72,8 @@ private extension AES256 {
   /// - Returns: random data
   static func randomData(count: Int) -> Data {
     var data = Data(count: count)
-    let status = data.withUnsafeMutableBytes {
-      SecRandomCopyBytes(kSecRandomDefault, count, $0)
+    let status = data.withUnsafeMutableBytes { ptr in
+        SecRandomCopyBytes(kSecRandomDefault, count, (ptr.baseAddress?.assumingMemoryBound(to: UnsafeRawBufferPointer.self))!)
```

Comment:
Should avoid force unwrapping. 

Also I see that this function is pretty much a duplicate of the one in `MedStrickerEncryptionTests`. Maybe we could extract this to a place where they can be reused.